hashes/src/util.rs

```diff
@@ -241,7 +241,7 @@ macro_rules! hash_newtype {
             }
 
             #[inline]
-            fn from_slice(sl: &[u8]) -> Result<$newtype, $crate::FromSliceError> {
+            fn from_slice(sl: &[u8]) -> $crate::_export::_core::result::Result<$newtype, $crate::FromSliceError> {
```

Comment:
Why is this one different?

Reply:
I noticed pretty much every other `macro_rules`  in this file was using this pattern, so I did the same to keep it consistent.